pkg/kudoctl/cmd/init/manager.go

```diff
@@ -59,19 +59,19 @@ func NewOptions(v string) Options {
 // Install uses Kubernetes client to install KUDO.
 func Install(client *kube.Client, opts Options, crdOnly bool) error {
 
-	clog.V(4).Printf("installing crds")
+	clog.Printf("✓ installing crds")
```

Comment:
personally I prefer a more quiet cli... I really preferred the verbosity at 4 rather than at 0.   I do like the unicode chars!

Reply:
Unicode ftw!

Reply:
I actually find it weird to run a command and have no feedback on what actually happened and if something happened. I think Zain or some of the SEs complained about it as well (that they don't have any feedback after the command is run). So I think we need to have at least a very short output for every command